### ISSUE TITLE
Follow TF performance guide in models available here

### ISSUE THREAD

**Issue by taion** [tensorflow/models#1598]
### System information
N/A

### Describe the problem
Currently, none of the models available here generally follow the TF performance guidelines. None of the models here support running in NCHW, and only the newest object detection models use the fused batch norm op.

As such, this makes the [published TF benchmarks](https://www.tensorflow.org/performance/benchmarks) somewhat misleading – while those benchmarks run at competitive speeds, the actual reference model code published here does not, and instead takes a very large performance hit.

Ideally the first-party example code here should all support NCHW and use fused batch norm where relevant, such that the performance obtained here is actually comparable with the claims made via the posted benchmarks.

**Comment by tfboyd** [comment]
The examples on tf.org are being updated and is something we are working on.  The code in tensorflow/models will be updated selectively and is owned by the individuals that posted the code.  I plan to make some improvements to the slim examples with help, but we found even just turning on fused batch_norm results in the original checkpoints not loading.

We are very shortly releasing a new area with models that are more thoroughly tested (also benchmarked to watch for specific regressions in the examples) and will be maintained by the TF team and hopefully the community.  This area will start out small and most likely will not excite you de to the small number of starting models.  I want to be clear about this because I do not want you to get excited.  On the plus side it will have canonical TF examples and provide better guidance for multi-GPU.  

I also have agreement from the team to add variable management (replicated, parameter server, NCCL, etc) as part of the API as well as add an easier way/pattern to do multi-GPU.  

If you have specific examples you think we should be updated please let me know so I can try to ensure they are updated first.  If your needs/wants match what I thought we would do I can try to accelerate the effort.  I can also be honest with you regarding plans for the examples or models that interest you. 

I am long winded but I ways appreciate and very much respect your feedback @taion

**Comment by taion** [comment]
That sounds great, thank you! I just wanted to check that people were aware of the issue. Given that this is already being taken care of, then I'm going to close this issue out for now, since it appears there's nothing specific to this issue to track.

**Comment by tfboyd** [comment]
If there are models people want updated the best way to get action is to open a issue with the model or directory.  There is hope we will have a magic tool in the near future that will actually do the conversion for you (no code change just a graph rewrite) and get almost the same performance as if you did it yourself.  In a dream world it might actually get better performance by knowing when NCHW is not the best choice even on GPU and determining if a transform is worth it.  I hope they have a prototype in the near future.  I t is always a pleasure Jimmy.  I hope to see you in person some day.
